models.yml

```diff
@@ -8,18 +8,25 @@
 #       At the moment we support only X == 6.
 #     - timestamp: Datetime as a unix timestamp. Why a number? This enables queries
 #       in the DB. And we do not need more precision than 1 second.
-#     - <T>[]: This indicates and arbitrary array of the given type. At the moment
+#     - <T>[]: This indicates an arbitrary array of the given type. At the moment
 #       we support only some types. You can add JSON Schema properties for items
 #       using the extra property `items`
 #     - color: string that must match ^#[0-9a-f]{6}$
 # Relations:
 #     - We have the following types: `relation`, `relation-list`, `generic-relation`
-#       and `generic-relation-list`.
+#       and `generic-relation-list`. All relation-types with a `list`-suffix are **always** realized as sql-statements,
+#       in case of n:m-relations with an intermediary table. 
 #     - Non-generic relations: The simple syntax for such a field
-#       `to: <collection>/<field>`. This is a reference to a collection. The reverse
-#       relation field in this collection is <field>. E. g. in a motion the field `meeting_id`
-#       `reference: <collection(field1, ...)` This can be used, if the foreign key reference
-#        is not the <collection/"id">, where the collection is the same as that from the `to`.
+#       `to: <collection>/<field>`. This is a reference to a collection-field, the "other" side of a relation.
+#           In the relational DB this will be a field in a view filled by an automatically generated sql.
+#           During the development of the relational db one relation can have a `to` and a `reference` property.
+#           In this case the `to` value is only used for historical reason, onky the `reference` property is used
+#           for the schema generation.  
+#           If necessary the sql can be written manually, see `sql`.
+#       `reference: <collection(field1, ...)` This will create a foreign key constraint. Therefore a `reference` is
+#           the indicator for a real field in the table and you cannot use it for relation types with a `list` suffix.
```

Comment:
Also, this is not the case in many places throughout the `model.yml`

Reply:
> Also, this is not the case in many places throughout the `model.yml`

Yes, I told you and this will change